content/guides/tasks/assignments/unassign.md

```diff
@@ -0,0 +1,15 @@
+---
+rank: 3
+related_endpoints:
+  - delete_task_assignments_id
+related_guides: []
+required_guides: []
+alias_paths: []
+---
+
+# Unassign a task
+
+To unassign a task, you will need to provide the Box API with the `id` of the
+assigned task.
```

Comment:
I think it's technically the ID of the task assignment, not the task itself. Right?

Reply:
you're right. I blanked a word.